site/src/app/v2/trouver-un-club/[club-name]/page.tsx

```diff
@@ -1,19 +1,26 @@
-'use client';
-
 import ClubDetails from '@/app/v2/trouver-un-club/[club-name]/components/clubDetails/ClubDetails';
 import EligibilityTestBanner from '../../../../../components/eligibility-test-banner/EligibilityTestBanner';
 import SocialMediaPanel from '../../../components/social-media-panel/SocialMediaPanel';
-import { useIsProVersion } from '../../../hooks/use-is-pro-version';
+import { Metadata } from 'next';
+
+interface Props {
+  params: { 'club-name': string };
+}
+
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+  return {
+    title: `Trouver un club partenaire - ${decodeURIComponent(params['club-name'])} - Pass Sport`,
+  };
+}
 
 const ClubPage = ({ params }: { params: { 'club-name': string } }) => {
   const clubName = decodeURIComponent(params['club-name']);
-  const isProVersion = useIsProVersion();
 
   return (
     <>
       <ClubDetails clubName={clubName} />
-      {!isProVersion && <EligibilityTestBanner />}
-      <SocialMediaPanel isProVersion={isProVersion} />
+      <EligibilityTestBanner />
+      <SocialMediaPanel isProVersion={false} />
```

Comment:
Pas beosin, c'est false par defaut
```suggestion
      <SocialMediaPanel>
```